packages/client/types/image.ts

```diff
@@ -0,0 +1,3 @@
+export interface TImage extends React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> {
+  borderRadius?: string;
+}
```

Comment:
Usually, practice is when we use `interface` then the prefix should be `I`, when we use `type` prefix should be `T`.